doc/API.md

```diff
@@ -232,13 +237,52 @@ This objects represents the server connection. You can create new ```JitsiConnec
         - recordingType - the type of recording to be used
         - callStatsID - callstats credentials
         - callStatsSecret - callstats credentials
+        - focusUserJid - The real JID of focus participant - can be overridden here
         - enableTalkWhileMuted - boolean property. Enables/disables talk while muted detection, by default the value is false/disabled.
+        - enableNoAudioDetection
+        - enableNoisyMicDetection
         - ignoreStartMuted - ignores start muted events coming from jicofo.
         - startSilent - enables silent mode, will mark audio as inactive will not send/receive audio
         - confID - Used for statistics to identify conference, if tenants are supported will contain tenant and the non lower case variant for the room name.
         - siteID - (optional) Used for statistics to identify the site where the user is coming from, if tenants are supported it will contain a unique identifier for that tenant. If not provided, the value will be infered from confID
         - statisticsId - The id to be used as stats instead of default callStatsUsername.
         - statisticsDisplayName - The display name to be used for stats, used for callstats.
+        - enableRemb
+        - enableTcc
+        - useRoomAsSharedDocumentName
+        - channelLastN
+        - startBitrate
+        - stereo
+        - forceJVB121Ratio - "Math.random() < forceJVB121Ratio" will determine whether a 2 people conference should be moved to the JVB instead of P2P. The decision is made on the responder side, after ICE succeeds on the P2P connection.
+        - hiddenDomain
+        - startAudioMuted
+        - startVideoMuted
+        - enableLayerSuspension - if set to 'true', we will cap the video send bitrate when we are told we have not been selected by any endpoints (and therefore the non-thumbnail streams are not in use).
+        - deploymentInfo
+            - shard
+            - userRegion
+        - p2p - Peer to peer related options
+            - enabled - enables or disable peer-to-peer connection, if disabled all conferences or going through the videobridge
```

Comment:
Modification - if disabled all media will be routed through the Jitsi Videobridge.